otx/cli/tools/train.py

```diff
@@ -210,7 +210,10 @@ def main():  # pylint: disable=too-many-branches
     save_model_data(output_model, args.save_model_to)
     print(f"[*] Save Model to: {args.save_model_to}")
 
-    if config_manager.data_config["val_subset"]["data_root"]:
+    if (
+        config_manager.data_config["val_subset"]["data_root"]
+        and hyper_parameters.algo_backend.train_type.value != "SELFSUPERVISED"
```

Comment:
Is there any case where `hyper_parameters.algo_backend.train_type.value` is not read properly? 
and
Is there any other way to give this as None inside the `ConfigManager`? I'm confused about the case of giving `val-data-roots` in self-sl.

Reply:
If there is any other way to set None to `config_manager.data_config["val_subset"]["data_root"]` during self-sl, it doesn't matter.
If there is the demand to use data.yaml created from `otx build` of supervised learning, there is `val-data-root` and it will be set. Self-sl doesn't use it, but unnecessary evaluation after training will be conducted.